opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -601,7 +600,7 @@ const inputResolveRefs = async (context, user, input, type, entitySetting) => {
           fetchingIds.push(...elements);
           expectedIds.push(...elements.map((e) => e.id));
         } else { // Single
-          if (dst === INPUT_FROM && isStixRefRelationship(type)) {
+          if (dst === 'from' && isStixRefRelationship(type)) {
```

Comment:
this one was tricky.

"from" is a key used in 2 different context: it's the "from" of a relationship, and the "from" in an email message.

`INPUT_FROM` was defined for use with email message only, but it was also imported and used in places related to relationship. 

Now that we renamed for email to `emailFrom` it's not working anymore for relationships. But it was never meant to work like that in the first place.

Reply:
Now that email is now on INPUT_EMAIL_FROM, are you sure is still needed to not used the constant?

Reply:
Most of the time the constant INPUT_NAME was not used, its' the string "from". 
It was an _input_ for email, but as for relationship it's not in any schema def, it's not an input.
We think it's an old confusion (email 'from' being used in place where we actually meant "from" of a relationship).

We removed the constant INPUT_NAME. It's not used in any schema def.

Reply:
This part was causing the test "should all elements correctly deleted" in middleware-test to fail, because the "from" was not resolved for one of the entity (it was a stix ref relationship). and yes we think "INPUT_FROM" was used by mistake because it was "from" until we changed it to "emailFrom", since it was an input specific to email message. And it was the only place where it was used as a constant, in all other places in middleware we use the string 'from' directly.